shared/pingpong/accounts.go

```diff
@@ -228,12 +251,15 @@ func prepareAssets(accounts map[string]uint64, client libgoal.Client, cfg PpConf
 					if !cfg.Quiet {
 						fmt.Printf("Opting in assets to account %v \n", addr2)
 					}
-					_, addrErr2 := client.AccountInformation(addr2)
+					acct, addrErr2 := client.AccountInformation(addr2)
 					if addrErr2 != nil {
 						fmt.Printf("Cannot lookup optin account\n")
 						err = addrErr2
 						return
 					}
+					if len(acct.Assets) >= proto.MaxAssetsPerAccount {
```

Comment:
Make the receiving address loop the outer loop, so that once the limit is reached, it can continue to the next address, instead of coming here every time while the limit is reached. 
In other words, for each "addr", for each "k" in addr, addr2 that reached the limit, will visit this line and break.

Reply:
completely removed 3 nested looks and do only two - on assets and on accounts

Reply:
well, it actually does not matter, same complexity